### PR TITLE
TELCODOCS-1920: Updating RDS for 4.16

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-1920](https://issues.redhat.com//browse/TELCODOCS-1920): Updating RDS for 4.16

Version(s):
Main and 4.16

Issue:
https://issues.redhat.com//browse/TELCODOCS-1920

Link to docs preview:
https://78553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-design-spec

QE review:
- [x] QE has approved this change.

Additional information:
Note to docs peer reviewers, all of the YAML changes are automatically generated. So no review required for any YAML changes in the snippets folder.

### CODE REVIEW

File: modules/telco-ran-yaml-ref-day-2-operators.adoc [hunk 1]
```diff
@@ -76,6 +76,35 @@ include::snippets/ztp_StorageOperGroup.yaml[]
 include::snippets/ztp_StorageSubscription.yaml[]
 ----
 
+[id="ztp-imagebasedupgrade-yaml"]
+.ImageBasedUpgrade.yaml
+[source,yaml]
+----
+include::snippets/ztp_ImageBasedUpgrade.yaml[]
+----
+////
```
Comment (rohennes) [inline modules/telco-ran-yaml-ref-day-2-operators.adoc:85]: Commenting LCA snippets out for the moment as I don't know if they should be included

File: modules/telco-core-crs-networking.adoc [hunk 2]
```diff
@@ -16,14 +16,19 @@ Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#tel
...
 Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-bgp-peer-yaml[bgp-peer.yaml],No,No
+Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-community-yaml[community.yaml],No,Yes
 Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-metallb-yaml[metallb.yaml],No,No
 Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-metallbns-yaml[metallbNS.yaml],Yes,No
 Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-metallbopergroup-yaml[metallbOperGroup.yaml],Yes,No
 Load balancer,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-metallbsubscription-yaml[metallbSubscription.yaml],No,No
 Multus - Tap CNI for rootless DPDK pod,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-mc_rootless_pods_selinux-yaml[mc_rootless_pods_selinux.yaml],No,No
+NMState Operator,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-nmstate-yaml[NMState.yaml],No,Yes
```
Comment (rohennes) [inline modules/telco-core-crs-networking.adoc:25]: @imiller0 - is the NMState Operator required?

**Review by sheriff-rh — APPROVED** [review]
Changes look good, Ronan - nice work!

**Comment by yogeshahiray** [comment]
LGTM from RDS testing perspective

**Comment by yprokule** [comment]
source-crs for the _RDS Core_ looks good

/lgtm

**Comment by jab-rh** [comment]
@rohennes, this oddly doesn't merge into 4.17 or 4.16.

cc @bergerhoffer

**Comment by bergerhoffer** [comment]
@rohennes @jab-rh I investigated why it's not picking cleanly, and it looks to be because someone had previously done a manual cherry pick and made changes in this file differently in main vs. 4.16.

So at least it doesn't look like there were any forgotten PRs here. 

Because this PR is just deleting that file entirely, I think you are safe to move forward with the manual cherry picks for 4.16 and 4.17. Thanks for checking first!

**Comment by rohennes** [comment]
Hi @jab-rh - I've created the manual CPs to 4.16 and 4.17 and resolved the conflicts. Thanks for your help!

**Pull request merged**